subprojects/core/src/integTest/groovy/org/gradle/api/file/ProjectLayoutIntegrationTest.groovy

```diff
@@ -217,7 +217,7 @@ class ProjectLayoutIntegrationTest extends AbstractIntegrationSpec {
         'FileCollection'             | 'Closure'        | 'project.layout.filesFor({ "%s/src/resource/file.txt" })'
         'FileCollection'             | 'List'           | 'project.layout.filesFor([ "%s/src/resource/file.txt" ])'
         'FileCollection'             | 'array'          | 'project.layout.filesFor([ "%s/src/resource/file.txt" ] as Object[])'
-        'FileCollection'             | 'FileCollection' | 'project.layout.filesFor(new org.gradle.api.internal.file.collections.SimpleFileCollection(new File("%s/src/resource/file.txt")))'
+        'FileCollection'             | 'FileCollection' | 'project.layout.filesFor(new org.gradle.api.internal.file.collections.FileCollectionAdapter(new org.gradle.api.internal.file.collections.ListBackedFileSet(new File("%s/src/resource/file.txt"))))'
```

Comment:
Why not use `ImmutableFileCollection.of()` here?

Reply:
good point.  i was in the mindset from updating the `ImmutableFileCollectionTest` to *not* use `ImmutableFileCollection`, but is perfectly reasonable here 😃